about/strategy.md

```diff
@@ -39,15 +39,15 @@ As a start, we wish to launch a **Managed JupyterHubs** service, and will begin
 - 2i2c manages JupyterHubs for at least two institutions.
 - 2i2c manages more lightweight, community-specific JupyterHubs for several smaller groups in research and education.
 - 2i2c manages a "generic" JupyterHub that is not tied to any single institution or group.
-- 2i2c has a beta-level business model for the first iteration of our Managed JupyterHub service.
+- 2i2c has a beta-level sustainability model for the first iteration of our Collaborative JupyterHub Service.
```

Comment:
IMO Managed is a more straightforward and broadly accepted term for what we do, while Collaborative is generally overloaded to the point of being a buzzword. I'd suggest we keep calling it 'Managed'